recipes/longphase/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "longphase" %}
+{% set version = "1.7.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('longphase', max_pin="x") }}
+
+source:
+  url: https://github.com/twolinin/longphase/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 58829b55ca55d796b836743466bc7b6caf82cfe9b88c25112bff6a0cd0353345
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

https://github.com/twolinin/longphase/blob/main/Makefile
You need both CC and CXX
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```